### ISSUE TITLE
animationEnabled is invalid

### ISSUE THREAD

**Issue by duguying** [aksonov/react-native-router-flux#2420]
### Version
Tell us which versions you are using: 

- react-native-router-flux v4.0.0-beta.21
- react-native v0.48.3

### Expected behaviour
page redirect without animation

### Actual behaviour
animate as right-to-left sweep

### Steps to reproduce
For non-obvious bugs, please fork this component, modify Example project to reproduce your issue and include link here.
```jsx
<Scene key="root" hideNavBar={true} animationEnabled={'false'}>
    <Scene key="login" component={LoginPage} hideNavBar={true} duration={0} animationEnabled={'false'} />
    <Scene key="login_form" component={LoginFormPage} hideNavBar={true} duration={0} animationEnabled={'false'} />
</Scene>
```

**Comment by mcabs3** [comment]
You are correct that this property doesn't exist anymore in V4. I will label this as an enhancement to add this feature back. Feel free to make a PR if you have time!

**Comment by esutton** [comment]
I would like to see no animation on tab change working.

I received a red box error on Android after adding:
````
animationEnabled={false)
````
Using:
````
		"react": "^16.0.0",
		"react-native": "^0.50.3",
		"react-native-router-flux": "^4.0.0-beta.24",
````
